src/LogInForm/LogInForm.js

```diff
@@ -3,7 +3,8 @@ import Map from '../Map/Map'
 import  { NavLink } from 'react-router-dom';
 import './LogInForm.css'
 
-const LogInForm = ({ reliefCenter, selectCenter, isCenterSelected }) => {
```

Comment:
Ah, I see where the conflict is coming from - it seems like it's because reliefCenter has been removed. The merge conflict highlights the whole form, but I think this is essentially it.